runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -211,8 +210,8 @@
 
   @VisibleForTesting static final int GCS_UPLOAD_BUFFER_SIZE_BYTES_DEFAULT = 1024 * 1024;
 
-  @VisibleForTesting static final String PIPELINE_FILE_FORMAT = "pipeline-%s.pb";
-  @VisibleForTesting static final String DATAFLOW_GRAPH_FILE_FORMAT = "dataflow_graph-%s.json";
+  @VisibleForTesting static final String PIPELINE_FILE_NAME = "pipeline.pb";
```

Comment:
Seems like this is just reverting to old behavior. Why would these names being unique an issue ? Did the change break Dataflow somehow ?

Reply:
In the previous PR, `forBytesToStage` only respected the target name as is so we needed to make the target name unique. Now `forBytesToStage` generates unique names by itself so we don't need to put UUID suffix manually.